gravitee-apim-e2e/tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "target": "ES2020",
     "lib": ["ES2020", "dom"],
     "types": ["./types", "cypress", "node", "node-fetch", "jsonwebtoken"],
-    "baseUrl": "./cypress",
+    "baseUrl": "./ui-test",
```

Comment:
🤔 should be `.` no api-test also need tsconfig no ?

Reply:
> What do you think about reviewing the npm scripts too ?
> 
> ```
>         "test": "ui-test open --config-file ui-test/integration/cypress-config.json",
>         "test:apim": "ui-test open --config-file ui-test/integration/apim/cypress-config.json",
>         "test:apim:bulk": "ui-test run --config-file ui-test/integration/apim/bulk/cypress-config.json",
>         "test:apim:rest-api": "ui-test run --config-file ui-test/integration/apim/rest-api/cypress-config.json",
>         "test:apim:e2e": "ui-test open --config-file ui-test/integration/apim/e2e/cypress-config.json",
>         "test:apim:ui": "ui-test open --config-file ui-test/integration/apim/ui/cypress-config.json",
>         "ci:apim:ui": "ui-test run --config-file ui-test/integration/apim/ui/cypress-config.json",
>         "test:apim:dev": "ui-test run --env printLogsToConsole=always --config-file ui-test/integration/apim/rest-api/cypress-config.json",
>         "test:apim:parallel": "ui-test run --record --group apis --spec 'ui-test/integration/apim/rest-api/apis/**/*' --config-file ui-test/integration/apim/rest-api/cypress-config.json",
>         "test:platform": "ui-test open --config-file ui-test/integration/platform/cypress-config.json",
>         "test:api:management": "jest api-test/management",
>         "test:ci": "jest api-test",
> ```
> 
> I wonder if we could have a little less xD and otherwise separate them like : `test-ui:...` && `test-api:...`

Agree. but each task is linked to a cypress config. 
I can rename now, and remove the useless tasks during the migration.
WDYT ?